tests/ui/transmutability/abstraction/abstracted_assume.rs

```diff
@@ -25,15 +25,15 @@ mod assert {
     {}
 }
 
-fn direct() {
+fn direct<T>() {
```

Comment:
I don't think you need to modify these existing tests. Maybe add new ones?

Reply:
I re-thought about it, I don't this case needs to be tested actually. So I'm just going to remove this change.